prolog/scasp/html.pl

```diff
@@ -8,8 +8,10 @@
           ]).
 :- use_module(common).
 :- use_module(output).
+:- use_module(dyncall).
```

Comment:
This one leads to a redefinition warning for not/1.    The reason isn't entirely clear to me.   We need to redefine not/1, but we can only do so if it is not yet imported into the current context.   It is redefined in embed.pl and then reexported to various modules.   This seems to go wrong on a cyclic import.   Anyway, as this library does not resolve anything for this module I removed the import.